odxtools/units.py

```diff
@@ -114,13 +114,13 @@ class Unit:
     def __post_init__(self):
         self._physical_dimension = None
 
-        if self.factor_si_to_unit is not None or self.offset_si_to_unit is not None or self.physical_dimension_ref is not None:
-            if self.factor_si_to_unit is not None and self.offset_si_to_unit is not None and self.physical_dimension_ref is not None:
-                warnings.warn(
-                    f"Error 54: If one of factor_si_to_unit, offset_si_to_unit and physical_dimension_ref is defined,"
-                    f" all of them must be defined: {self.factor_si_to_unit} and {self.offset_si_to_unit} and {self.physical_dimension_ref}",
-                    OdxWarning
-                )
+        #if self.factor_si_to_unit is not None or self.offset_si_to_unit is not None or self.physical_dimension_ref is not None:
+        #    if self.factor_si_to_unit is None or self.offset_si_to_unit is None or self.physical_dimension_ref is None:
```

Comment:
@andlaus commented out code?

Reply:
yeah, I wanted your opinion before removing it outright...

Reply:
ok, removed...

Reply:
It does not affect the encoding or decoding, so I don't think we actually need this check, seems like an overkill.